tools/clang/test/SemaHLSL/hlsl/objects/HitObject/hitobject-target-profile-error.hlsl

```diff
@@ -0,0 +1,9 @@
+// RUN: not %dxc -T lib_6_8 %s 2>&1 | FileCheck %s
+
+namespace dx {}
```

Comment:
What happens if you remove this? That would be the more common situation.

Reply:
Kept this test - but added another one that specifically checks that a namespace 'dx' is not predeclared pre-SM6.9